server/index.php

```diff
@@ -62,26 +62,21 @@
   } else {
       if (!empty($_GET["room"])) {
         switch ($_GET["room"]) {
-            case 'ballroom-de':
             case 'ballroom-a':
             case 'ballroom-b':
             case 'ballroom-c':
+            case 'ballroom-de':
             case 'ballroom-f':
             case 'ballroom-g':
             case 'ballroom-h':
-            case 'ballroom-gh':
-            case 'ballroom-i':
-            case 'ballroom-j':
             case 'room-101':
             case 'room-103':
-            case 'room-104':
+            case 'room-105':
```

Comment:
We were missing this in the `index.php` so when you when to `http://localhost/?room=room-105&year=2024&month=3&day=16&hour=11&minute=30` the it would show the rooms signage but instead the default (full schedule) signage.

Setting this here fix it